### DISCUSSION TITLE
YouTube opening and closing Android 13 and 14

### DISCUSSION THREAD

**Discussion by Frontline11** [ReVanced/revanced-discussions#1019] [General]
Hello, I and many people are having a problem with YouTube ReVanced 18.45.41 Patch 1.99.0 YouTube keeps opening and closing with Android 13 and Android 14, I've already done several tests disabling some Patches to see if there's a problem but it didn't work , And it keeps opening and closing, I ask you to check that microG is not causing this problem in your accounts

**Comment by vividhues** [comment]
If you're using Microg-RE then downgrade to version 3.4

**Reply by Frontline11** [reply]
I'm using the original microG that the developers ask to use those old micro G still 0.24

**Reply by vividhues** [reply]
try using different microg then. I had a similar issue and it turned out to be microg

**Reply by Frontline11** [reply]
please give me this MicroG that you used and solved the problem, please give me this microG please , I use Android 13 and Android 14

**Reply by vividhues** [reply]
https://github.com/WSTxda/MicroG-RE/releases/tag/3.4 
this is the one I used.

**Reply by Frontline11** [reply]
Thanks friend I downloaded microG -RE V8

**Reply by Frontline11** [reply]
microG Universal fits everyone, armv7, aRMV 8 ?

**Reply by vividhues** [reply]
yes

**Reply by Frontline11** [reply]
Ok excellent, thank you very much

**Comment by LisoUseInAIKyrios** [comment]
A new MicroG exists, and the advice here is no longer up to date.
